x/auth/types/params.go

```diff
@@ -54,7 +54,7 @@ func ParamKeyTable() paramtypes.KeyTable {
 func (p *Params) ParamSetPairs() paramtypes.ParamSetPairs {
 
 	return paramtypes.ParamSetPairs{
-		paramtypes.NewParamSetPair(KeyMaxMemoCharacters, p.MaxMemoCharacters, validateMaxMemoCharacters),
+		paramtypes.NewParamSetPair(KeyMaxMemoCharacters, &pt.UInt64Value{Value: p.MaxMemoCharacters}, validateMaxMemoCharacters),
```

Comment:
@alexanderbez this is the kind of stuff we would need all over the place for param values if we wanted to keep the proto representation as primitive types.

Reply:
Yup, makes sense!